behave_django/runner.py

```diff
@@ -0,0 +1,62 @@
+try:
+    from django.test.runner import DiscoverRunner
+except ImportError:
+    from django.test.simple import DjangoTestSuiteRunner as DiscoverRunner
+
+from django.core.management import call_command
+
+try:
+    from django.shortcuts import resolve_url
+except ImportError:
+    # support Django 1.4, which has no resolve_url() yet
+    from django.shortcuts import redirect
+
+    resolve_url = lambda to, *args, **kwargs: \
+        redirect(to, *args, **kwargs)['Location']
+
+from behave_django.testcase import (BehaviorDrivenTestCase,
+                                    ExistingDatabaseTestCase)
+
+
+class BehaviorDrivenTestRunner(DiscoverRunner):
+    testcase_class = BehaviorDrivenTestCase
+
+    def before_scenario(self, context):
+        context.test = self.testcase_class()
+        context.test.setUpClass()
+        context.test()
+
+        context.base_url = context.test.live_server_url
+
+        def get_url(to=None, *args, **kwargs):
+            return context.base_url + (
+                resolve_url(to, *args, **kwargs) if to else '')
+
+        context.get_url = get_url
+
+    def load_fixtures(self, context):
```

Comment:
This was necessary to allow fixtures be defined in `environment.before_scenario()` and loaded *after* `before_scenario()` was completed. Also, because the integration of django into behave needs to run *before* `environment.before_scenario()` is reached. The documentation on factories supported that point too.